index.html

```diff
@@ -165,8 +165,34 @@ <h4>Skills</h4>
             <li>Gitlab</li>
             <li>Terminal</li>
         </ul>
-
+    </section>
+    <section class="content">
+        <h3>
+            I'm always interested in hearing about new projects, so if you'd 
+            like to chat please get in touch.
+        </h3>
     </section>
 
+    <form action="https://formspree.io/f/xdobraqv" method="post" class="form1">
+        <input name="input1" type="text" required class="name" placeholder=" Full name" maxlength="30">
+        <div class="div-intermediate1"></div>
+        <input name="input2" type="email" required class="email" placeholder="Email address">
+        <div class="div-intermediate2"></div>
+        <textarea name="text-area" required id="#" cols="30" rows="10" class="info" placeholder=" Enter text here"></textarea>
+        <button class="button4" type="button">Get in touch</button>
+    </form>
```

Comment:
- [ ] `required`
- [ ] Good job with the form so far :clap: :clap: 
- [ ] you missed out a bit style on the figma please see screenshot below

`Your Template`
![Screenshot from 2022-07-05 10-49-36](https://user-images.githubusercontent.com/66230306/177424864-f813fe7b-8161-4ef4-b9b3-96ee044899a3.png)

`Figma Template`
![Screenshot from 2022-07-05 10-48-42](https://user-images.githubusercontent.com/66230306/177424965-41cf3ce9-bca0-4e24-af16-eeae228ba549.png)